media/main.js

```diff
@@ -94,13 +93,12 @@ function () {
 					let green = "green";
 					let red = "red";
 					if (String(isAvailable).toLowerCase() === 'true') {
-						requirementMessage = 'Available';
 						element.style.color = green;
+						element.textContent = `Status: Available`;
```

Comment:
given that there is no interpolation, simple-quote can be used instead fo backtick.